lib/rubocop/cop/cop.rb

```diff
@@ -212,10 +217,16 @@ def reference_url
         (url.nil? || url.empty?) ? nil : url
       end
 
+      def details
+        details = cop_config && cop_config['Details']
+        (details.nil? || details.empty?) ? 'No extra details set.' : details
```

Comment:
Yeah, I just put that in there so I could actually display something - still not sure precisely how to test this